app/controllers/step_by_step_pages_controller.rb

```diff
@@ -124,7 +124,7 @@ def unpublish
 
   def unschedule
     set_current_page_as_step_by_step
-    @step_by_step_page.update(scheduled_at: nil)
+    @step_by_step_page.update(scheduled_at: nil, update_type: nil, public_change_note: nil)
```

Comment:
I think these updates can be added to the `mark_as_unscheduled` method in the model.